src/main/java/codeu/controller/ActivityServletTest.java

```diff
@@ -0,0 +1,70 @@
+// Copyright 2017 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package codeu.controller;
+
+import codeu.model.data.User;
+import codeu.model.store.basic.UserStore;
+import java.io.IOException;
+import java.time.Instant;
+import java.util.UUID;
+
+import javax.servlet.RequestDispatcher;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mockito;
+
+public class ActivityFeedServletTest {
+
+  private ActivityFeedServlet activityFeedServlet;
+  private HttpServletRequest mockRequest;
+  private HttpSession mockSession;
+  private HttpServletResponse mockResponse;
+
+  @Before
+  public void setup() {
+    activityFeedServlet = new ActivityFeedServletA;
+
+    mockRequest = Mockito.mock(HttpServletRequest.class);
+    mockResponse = Mockito.mock(HttpServletResponse.class);
+
+    mockSession = Mockito.mock(HttpSession.class);
+    Mockito.when(mockRequest.getSession()).thenReturn(mockSession);
+
+    mockRequestDispatcher = Mockito.mock(RequestDispatcher.class);
+  }
+
+  @Test
+  public void testDoGet_friendUsername() throws IOException, ServletException {
+    Mockito.when(mockSession.getAttribute("user")).thenReturn("Welcome to the activity feed page");
```

Comment:
The "user" attribute usually contains the username of the client, not strings with spaces like the one used here.
Is this intentional?

Reply:
I changed it! Thanks for the catch